packages/atlas/src/views/viewer/EditMembershipView/EditMembershipView.tsx

```diff
@@ -100,7 +106,10 @@ export const EditMembershipView: React.FC = () => {
           secondaryButton={{
             visible: true,
             text: 'Cancel',
-            onClick: resetForm,
+            onClick: () => {
+              resetForm()
+              navigate(absoluteRoutes.viewer.member(activeMembership?.handle))
```

Comment:
go back where exactly?

Reply:
To where you came from. To get to this page you will click the `Edit profile` button in the membership header right? When you did that, you was on some membership view, so that's where I'm suggesting we go back. We can just also go to main member view to simplify, in that case let's use regular navigation instead of `navigate`

Reply:
Ah I see you already did that, let's leave it as it is then

Reply:
yeah I think thats best